docker/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN	rm -rf /usr/local/tomcat/webapps/* && \
 	chown -R tomcat /var/rdf4j /usr/local/tomcat && \
 	chmod a+x /usr/local/tomcat /usr/local/tomcat/bin /usr/local/tomcat/bin/catalina.sh
 
-COPY --from=temp /tmp/eclipse-rdf4j*/war/*.war /usr/local/tomcat/webapps
+COPY --from=temp /tmp/eclipse-rdf4j*/war/*.war /usr/local/tomcat/webapps/
```

Comment:
I'm not sure why this doesn't trip anyone else up when using this Dockerfile, but on my machine it consistently fails on this command when there's no trailing slash. I'm running Docker version 20.10.21, btw.

Reply:
Hmz indeed, according to the documentation it actually shouldn't work without the slash ...
FWIW: also using 20.10.21, on Debian